go_covert_lib/controller/channel/tcpNormal/tcpNormal_test.go

```diff
@@ -9,26 +9,6 @@ import (
 	"time"
 )
 
-// These tests must be run in sudo mode
```

Comment:
Why are we deleting all of this?

Reply:
@MichaelDysart  said the comments are not applicable in this test